redash/query_runner/pg.py

```diff
@@ -70,6 +70,11 @@ def configuration_schema(cls):
                 "dbname": {
                     "type": "string",
                     "title": "Database Name"
+                },
+                "sslmode": {
+                   "type": "string",
+                   "title": "SSL Mode",
+                   "default": "require"
```

Comment:
Meh. It doesn't modify any existing connections just sets the defaults for new ones. I can change it to prefer though.